libs/ng-utils/src/lib/use-http-image-source/use-http-image-source.pipe.ts

```diff
@@ -0,0 +1,77 @@
+import { HttpClient, HttpResponse } from '@angular/common/http';
+import { ChangeDetectorRef, Inject, OnDestroy, Pipe, PipeTransform } from '@angular/core';
+import { DomSanitizer, SafeUrl } from '@angular/platform-browser';
+import { BehaviorSubject, of, Subscription } from 'rxjs';
+import { catchError, distinctUntilChanged, filter, map, switchMap } from 'rxjs/operators';
+
+@Pipe({
+  name: 'useHttpImgSrc',
+  pure: false,
+})
+export class UseHttpImageSourcePipe implements PipeTransform, OnDestroy {
+  private subscription!: Subscription;
```

Comment:
What if we created an empty subscription here and only added to it inside of `ngOnInit`?
```suggestion
  private subscription = new Subscription()
```

Reply:
I found this: https://github.com/angular/angular/issues/10279

And I've checked the async pipe, and it turns out that Angular does not invoke ngOnInit in pipes, because pipes don't have `@Input()` properties

Reply:
I moved it to the constructor